Mathlib/CategoryTheory/Enriched/EnrichedCat.lean

```diff
@@ -0,0 +1,148 @@
+/-
+Copyright (c) 2025 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer
+-/
+import Mathlib.CategoryTheory.Enriched.Basic
+import Mathlib.CategoryTheory.Bicategory.Basic
+
+/-!
+# The bicategory of `V`-enriched categories
+
+We define the bicategory `EnrichedCat V` of (bundled) `V`-enriched categories for a fixed monoidal
+category `V`.
+
+## Future work
+
+* Define change of base and `ForgetEnrichment` as 2-functors.
+* Define the bicategory of enriched ordinary categories.
+-/
+
+
+universe w v u u₁ u₂
+
+namespace CategoryTheory
+
+open MonoidalCategory
+
+variable (V : Type v) [Category.{w} V] [MonoidalCategory V]
+
+/-- Category of `V`-enriched categories for a monoidal category `V`. -/
+def EnrichedCat := Bundled (EnrichedCategory.{w, v, u} V)
+
+namespace EnrichedCat
+
+instance : CoeSort (EnrichedCat V) (Type u) :=
+  ⟨Bundled.α⟩
+
+instance str (C : EnrichedCat.{w, v, u} V) : EnrichedCategory.{w, v, u} V C :=
+  Bundled.str C
+
+/-- Construct a bundled `EnrichedCat` from the underlying type and the typeclass. -/
+def of (C : Type u) [EnrichedCategory.{w} V C] : EnrichedCat.{w, v, u} V :=
+  Bundled.of C
+
+open EnrichedCategory ForgetEnrichment
+
+variable {C D E E' : EnrichedCat.{w, v, u} V}
+
+/-- Whisker a `V`-enriched natural transformation on the left. -/
+@[simps!]
+def whiskerLeft
+    (F : EnrichedFunctor V C D) {G H : EnrichedFunctor V D E} (α : G.forget ⟶ H.forget) :
+    (F.comp V G).forget ⟶ (F.comp V H).forget :=
```

Comment:
Here and below, I think it would be nicer if the signature could be 
```lean
def whiskerLeft (F : EnrichedFunctor V C D) {G H : EnrichedFunctor V D E} (α : G ⟶ H) :
    F.comp V G ⟶ F.comp V H :=
```
I think my suggestion about making homs a one-field struct might help with that.